detection-rules/attachment_malicious_onenote_commands.yml

```diff
@@ -0,0 +1,15 @@
+name: "Attachment: Malicious OneNote Commands"
+description: |
+  Scans for OneNote attachments that contain suspicious commands that may indicate malicious activity.
+references:
+  - "https://www.trustedsec.com/blog/new-attacks-old-tricks-how-onenote-malware-is-evolving/"
+type: "rule"
+severity: "medium"
+source: |
+type.inbound
+and any(attachments,(.file_extension == 'one')
+and any(beta.binexplode(.), any(.scan.strings.strings, strings.ilike(., "*WshShell*", "*ExecuteCmdAsync*", "*CreateObject*", "*Wscript.Shell*","*schtasks*", "*CreateProcess*","*winmgmts*", "*SetEnvironmentVariable*"))))
+tags: 
+  - "Suspicious attachment"
+  - "Malware"
+  - "Office exploit"
```

Comment:
```suggestion
```

Reply:
i think we'd call this an exploit if there was a vulnerability / potential for a CVE, but in this case it looks like it's a "feature" 😅